### DISCUSSION TITLE
How is the WeightedEnsemble model done for time series?

### DISCUSSION THREAD

**Discussion by gianluca-coletta** [autogluon/autogluon#3953] [Q&A] [answered]
Hi, I was wondering if it was possible to find the way the **WeightedEnsemble** model is done but I couldn't find documentation about it in the AutoGluonTS library. 

My main questions on the topic are as follows: 

- Does it simply do a weighted average of the models it considers? 
- How does it decide which models to choose and what weight to assign to them?
- Is there any documentation about this? 

Thank you all for your help and I hope my request was clear :)

**Comment by shchur** [comment, marked as answer]
We describe the ensembling method in the publication on AG-TS https://arxiv.org/pdf/2308.05566.pdf

**Reply by gianluca-coletta** [reply]
Ty so much @shchur!
